### PR TITLE
Closes #124: Fix remaining doctests

### LINKED ISSUE #124: Fix remaining doctests
@moreati Has fixed a large number of the doctests in #120, but apparently there are still some remaining. We should go through and fix or remove those, as appropriate.

### PR DESCRIPTION
**Pull request by rs2**
## Summary of changes

Fix remaining doctests.

Closes #124 

### Pull Request Checklist
- [x] Changes have tests
- [x] Authors have been added to [AUTHORS.md](https://github.com/dateutil/dateutil/blob/master/AUTHORS.md)
- [x] News fragment added in changelog.d. See [CONTRIBUTING.md](https://github.com/dateutil/dateutil/blob/master/CONTRIBUTING.md#changelog) for details

### CODE REVIEW

File: dateutil/tz/tz.py [hunk 1]
```diff
@@ -249,30 +246,33 @@ def _naive_is_dst(self, dt):
...
-        # >>> datetime.datetime(2003,2,15,23,tzinfo=t).tzname()
-        # 'BRST'
-        # >>> datetime.datetime(2003,2,15,22,tzinfo=t).tzname()
-        # 'BRDT'
-        # >>> datetime.datetime(2003,2,15,23,tzinfo=t).tzname()
-        # 'BRDT'
-        #
-        # Here is a more stable implementation:
-        #
+        """
```
[the remark this answers is not in the document]
Comment (pganssle) [reply dateutil/tz/tz.py]: This is a private method anyway, so this is basically a block comment. Shouldn't be rendered anywhere or run through the doctests I would assume.

File: dateutil/tz/tz.py [hunk 2]
```diff
@@ -1726,13 +1724,13 @@ def resolve_imaginary(dt):
...
-        >>> from datetime import datetime
+        >>> import datetime
         >>> NYC = tz.gettz('America/New_York')
-        >>> print(tz.resolve_imaginary(datetime(2017, 3, 12, 2, 30, tzinfo=NYC)))
+        >>> print(tz.resolve_imaginary(datetime.datetime(2017, 3, 12, 2, 30, tzinfo=NYC)))
         2017-03-12 03:30:00-04:00
 
         >>> KIR = tz.gettz('Pacific/Kiritimati')
-        >>> print(tz.resolve_imaginary(datetime(1995, 1, 1, 12, 30, tzinfo=KIR)))
+        >>> print(tz.resolve_imaginary(datetime.datetime(1995, 1, 1, 12, 30, tzinfo=KIR)))
```
Comment (pganssle) [inline dateutil/tz/tz.py]: I think I would prefer these and their like to be `from datetime import datetime`, but I don't feel strongly about this.

Comment (pganssle) [reply dateutil/tz/tz.py]: Also the correct thing to do is I think `1994, 12, 31` which should resolve to `1995, 1, 1`. This was a mistake in the tzdatabase fixed in 2018e.

**Comment by pganssle** [comment]
The failure on Python 3.3 is probably due to the latest version of `pytest` not supporting Python 3.3. At some point we'll drop support for Python 3.3, but for today maybe we'll just not run the doctests on 3.3. @gaborbernat Do you know the best way to do that selectively with tox with a minimum of DRY?

File: dateutil/tz/tz.py (hunk 2 above)
Comment (pganssle) [reply dateutil/tz/tz.py]: You can change this to:

```python
>>> APIA = tz.gettz('Pacific/Apia')
>>> print(tz.resolve_imaginary(datetime(2011, 12, 30, 12, tzinfo=APIA))
2011-12-31 12:00:00+14:00
```

**Comment by rs2** [comment]
@pganssle Over to you.

**Comment by rs2** [comment]
Addressed as a part of https://github.com/dateutil/dateutil/pull/769

**Pull request closed without merging**